### DISCUSSION TITLE
History V3 Directory structure

### DISCUSSION THREAD

**Discussion by simonblack** [erigontech/erigon#7822] [Q&A] [answered]
i started syncing history v3 a day ago and noticed today that even after the downloader is completed and on to the tx execution stage, everything is still operating in the snapshots (and temp) directories

Has the layout changed or will the chaindata folder be filled later on in the process? I currently have snapshots and temp directories mounted on lower performing drives, would need to switch things up if thats the case!

cheers

**Comment by AskAlexSharov** [comment, marked as answer]
history v3 is not ready for users preview yet.
there is used `datadir/recondb` for initial sync and later it will fill `datadir/chaindata`

**Reply by simonblack** [reply]
Thanks! If I may ask, does this mean breaking changes requiring a potential complete resync or just issues with stability? Im only spinning up a node to query the most recent block data and execute offline transactions

thanks

**Reply by simonblack** [reply]
Im trying history v3 because the v2 sync slowed down during the final block execution round (would bring my chain up to sync) and to a halt during the final account hashing stage (unless im misunderstanding the logs), judging by how long it took to increment from 0000000000010000 to 0000000000020000 (2 days) i decided to give v3 a shot after reading about the multithreading

im using nvme and its at about 50% capacity, everything was syncing fine as per the docs (about 2-4 days) until this stage

**Reply by AskAlexSharov** [reply]
Erigon3 (history v3) require re-sync. No migration path from Erigon2.
